week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,23 +32,38 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-
+  return house.currentOwner.firstName + " " + house.currentOwner.lastName + ".";
 }
 
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return [
+    kinningParkHouse.currentOwner.email,
+    " " + parkAvenueHouse.currentOwner.email
+  ];
 }
 
 // returns the address for the cheapest house out of the two
 function getCheapestAddress(house1, house2) {
-
+  if (kinningParkHouse.price < parkAvenueHouse.price) {

```

Comment:
There's a bug here.  This function should test the houses given in the parameters `house1`, and `house2`.  But it ALWAYS tests kinningParkHouse and parkAvenueHouse!